op-node/rollup/sync/start.go

```diff
@@ -109,6 +109,9 @@ func FindL2Heads(ctx context.Context, cfg *rollup.Config, l1 L1Chain, l2 L2Chain
 		return nil, fmt.Errorf("failed to fetch current L2 forkchoice state: %w", err)
 	}
 
+	lgr.Info("Running FindL2Heads", "unsafe", result.Unsafe, "safe", result.Safe, "finalized", result.Finalized,
```

Comment:
Maybe:
```suggestion
	lgr.Info("Loaded current L2 heads", "unsafe", result.Unsafe, "safe", result.Safe, "finalized", result.Finalized,
```
or something a bit more about what it's doing not just the method name. Happy for you to ignore me on this though. :)

Reply:
++